modules/glfw/src/filewatcher.cpp

```diff
@@ -29,21 +29,11 @@
 
 #include <modules/glfw/filewatcher.h>
 
-#include <inviwo/core/common/inviwoapplication.h>
-#include <inviwo/core/util/stringconversion.h>
-#include <inviwo/core/util/logcentral.h>
-#include <inviwo/core/util/zip.h>
-#include <inviwo/core/util/fileobserver.h>
-#include <inviwo/core/util/assertion.h>
-#include <inviwo/core/util/stdextensions.h>
-#include <inviwo/core/util/filesystem.h>
-#include <inviwo/core/util/threadutil.h>
-
-#include <array>
-#include <mutex>
-#include <atomic>
-#include <chrono>
-#include <algorithm>
+#include <inviwo/core/util/assertion.h>      // for IVW_ASSERT
+#include <inviwo/core/util/logcentral.h>     // for LogCentral, LogWarn
+#include <inviwo/core/util/stdextensions.h>  // for erase_remove
+
+#include <algorithm>  // for find
 
 #ifdef WIN32
```

Comment:
Build fails on Windows due to missing includes
mutex, chrono, thread, ...

Reply:
fixed